apps/web/src/app/(dashboard)/documents/data-table-action-dropdown.tsx

```diff
@@ -233,14 +233,12 @@ export const DataTableActionDropdown = ({ row, team }: DataTableActionDropdownPr
         onOpenChange={setMoveDialogOpen}
       />
 
-      {isDuplicateDialogOpen && (
```

Comment:
I think we may have been killing it with the conditional here for a reason, can you test what happens if you open and close the dialog and then pick a different row to duplicate?

Reply:
I don't think that will be a problem

![CleanShot 2025-01-21 at 07 22 29](https://github.com/user-attachments/assets/3831793b-4b7a-450a-bc66-58d2431854e1)